risc0/zkp/src/prove/accum.rs

```diff
@@ -38,6 +41,8 @@ impl<E: Elem> Accum<E> {
         }
     }
 
+    // Generates "pre-" side of grand product accumulation
+    // TODO check with Jeremy if this is accurate. (what is "prefix?")
```

Comment:
Here 'prefix' is related to prefix-product mechanism for doing parallel computations of products (see https://en.wikipedia.org/wiki/Prefix_sum, same idea since the underlying mechanism works for anything commutative and associative)